### PR TITLE
volume slider handle glow effect using system theme

### PR DESCRIPTION
**Pull request by narunlifescience**
with this modification volume slider glow effect can use system theme highlight color. If at some point, there comes a need to change the volume slider handle icon, this code should work there as well. 

untitled

### CODE REVIEW

File: src/widgets/sliderwidget.cpp [hunk 1]
```diff
@@ -366,3 +353,41 @@ QPixmap Amarok::VolumeSlider::volumePixmapDraw () const {
   // Return QPixmap
   return pixmap;
 }
+
+void Amarok::VolumeSlider::VolumeSliderHandle() {
```
Comment (ArnaudBienner) [inline src/widgets/sliderwidget.cpp]: Please use a more meaningful name (e.g. drawVolumeSliderHandle)

File: src/widgets/sliderwidget.h [hunk 2]
```diff
@@ -99,6 +99,7 @@ class VolumeSlider : public Slider {
  public:
   VolumeSlider(QWidget* parent, uint max = 0);
   QPixmap volumePixmapDraw() const;
+  void VolumeSliderHandle();
```
Comment (ArnaudBienner) [inline src/widgets/sliderwidget.h]: Should be private

File: src/widgets/sliderwidget.h [hunk 3]
```diff
@@ -99,6 +99,7 @@ class VolumeSlider : public Slider {
  public:
   VolumeSlider(QWidget* parent, uint max = 0);
   QPixmap volumePixmapDraw() const;
```
Comment (ArnaudBienner) [inline src/widgets/sliderwidget.h]: This one should be private too

File: src/widgets/sliderwidget.cpp (hunk 1 above)
Comment (narunlifescience) [inline src/widgets/sliderwidget.cpp]: Ok

File: src/widgets/sliderwidget.h (hunk 2 above)
Comment (narunlifescience) [inline src/widgets/sliderwidget.h]: Oh yes, they are accessed only inside public member function of the class.. I can change that

File: src/widgets/sliderwidget.h (hunk 3 above)
Comment (narunlifescience) [inline src/widgets/sliderwidget.h]: I can change that too..

**Comment by narunlifescience** [comment]
Done!

**Comment by ArnaudBienner** [comment]
@hatstand do you want to review this as well?

**Comment by narunlifescience** [comment]
Done!

**Pull request merged**